extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+## Contributors
+
+* Sean Lilley, Cesium
+* Peter Gagliardi, Cesium
+* Sam Suhag, Cesium
+* Patrick Cozzi, Cesium
+* Bao Tran, Cesium
+
+<!-- omit in toc -->
+## Status
+
+Draft
+
+<!-- omit in toc -->
+## Dependencies
+
+Written against the 3D Tiles 1.0 specification.
+
+Adds new functionality to the [`3DTILES_implicit_tiling` extension](../../3DTILES_implicit_tiling/0.0.0). See [Implicit Tile Metadata](#implicit-tile-metadata).
+
+<!-- omit in toc -->
+## Optional vs. Required
+
+This extension is optional, meaning it should be placed in the tileset JSON `extensionsUsed` list, but not in the `extensionsRequired` list.
+
+<!-- omit in toc -->
+## Contents
+
+- [Overview](#overview)
+- [Use Cases](#use-cases)
+- [Compatibility Notes](#compatibility-notes)
+- [Concepts](#concepts)
+  - [Schemas](#schemas)
+  - [Tileset Metadata](#tileset-metadata)
+  - [Tile Metadata](#tile-metadata)
+    - [Implicit Tile Metadata](#implicit-tile-metadata)
+  - [Group Metadata](#group-metadata)
+  - [Feature Metadata](#feature-metadata)
+  - [Statistics](#statistics)
+  - [Semantics](#semantics)
+  - [Styling](#styling)
+- [JSON Schema Reference](#json-schema-reference)
+
+## Overview
+
+This extension defines a standard mechanism for adding metadata to 3D Tiles. This includes:
+
+* Tileset metadata - metadata about the tileset as a whole.
+* Tile metadata - metadata about individual tiles.
+* Group metadata - metadata about groups of content.
+* Feature metadata - metadata about features. See the companion glTF extension [EXT_feature_metadata](https://github.com/CesiumGS/glTF/tree/master/extensions/2.0/Vendor/EXT_feature_metadata/1.0.0).
+
+<img src="figures/metadata-granularity.png"  alt="Metadata Granularity" width="600">
+
+A tileset defines a **schema**. A schema has a set of **classes** and **enums**. A class contains a set of **properties**, which may be numeric, boolean, string, enum, or array types.
+
+**Entities** (such as tiles, features, etc.) conform to classes and contain **property values**. Depending on the context, property values may be stored in JSON or binary.
+
+**Statistics** provide aggregate information about the metadata. For example, statistics may include the min/max values of a numeric property for mapping property values to color ramps with the [declarative styling language](../../../specification/Styling) or the number of enum occurrences for creating histograms.
+
+By default, properties do not have any inherent meaning. A property may be assigned a **semantic**, an identifier that describes how the property should be interpreted. The full list of built-in semantics can be found in the [Cesium Metadata Semantic Reference](../../../specification/Metadata/Semantics). Tileset authors may define their own application- or domain-specific semantics separately.
+
+This extension implements the [Cesium 3D Metadata Specification](../../../specification/Metadata/1.0.0), which describes the metadata format in full detail.
+
+## Use Cases
```

Comment:
A spec should answer why, before what and how.

Can we put more of this motivation up front in the intro?  Then the reader knows why this spec defines entities, statistics, etc. when they read the current intro.

[Start with Why](https://www.ted.com/talks/simon_sinek_how_great_leaders_inspire_action?language=en)